src/rules/no-collection-size-mischeck.ts

```diff
@@ -29,6 +29,13 @@ const CollectionSizeLike = ['length', 'size'];
 const rule: Rule.RuleModule = {
   meta: {
     type: 'problem',
+    docs: {
+      description: 'Collection sizes and array length comparisons should make sense',
+      category: 'Possible Errors',
+      recommended: 'error',
+      url: 'https://github.com/SonarSource/eslint-plugin-sonarjs/blob/master/docs/rules/no-collection-size-mischeck.md',
+      requiresTypeChecking: true,
```

Comment:
I don't find the doc for this property. Do you know where is it coming from?

Reply:
It's coming from typescript-eslint (see [Rule.ts](https://github.com/typescript-eslint/typescript-eslint/blob/f2c078cb1726349e5d1bb252e2653f4eb6310b68/packages/experimental-utils/src/ts-eslint/Rule.ts#L39)). After looking at how it is used, we don't need it.